Doc/library/re.rst

```diff
@@ -711,6 +715,9 @@ three digits in length.
    The :samp:`'\\N\\{{name}\\}'` escape sequence has been added. As in string literals,
    it expands to the named Unicode character (e.g. ``'\N{EM DASH}'``).
 
+.. versionchanged:: next
+   The ``'\z'`` escape sequence has been added.
```

Comment:
Perhaps move this directly under `\z`, like `\B` has a similar note up there.

Reply:
Indeed, these versionchanges were only applied to the escape sequences defined after the definition list.